src/common/Touchable.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Thanks for moving this out into a common component! Makes it much easier to maintain a consistent style :)